doc/api/api_changes/2017-02-06_TAC.rst

```diff
@@ -0,0 +1,11 @@
+Do not clip lw when scaling dashes
+``````````````````````````````````
+
+Slightly change the algorithm to no longer clip the scaling factor,
+thus the patterns will continue to shrink at thin line widths.  If the
+line width is smaller than the effective pixel size, this may result
+in dashed lines turning into solid gray lines.  This also required
+slightly tweaking the default patterns for '--', ':', and '.-' so that
+with the default line-width the final patterns would not change.
```

Comment:
line-width <- line width? (to be consistent with the other utterances of the expression).

Beside, it is not totally true that the new default patterns are exactly the same as before (e.g. the dotted pattern that is not a  50 %-50 % pattern anymore). Maybe the sentence should just state that the default patterns were tweaked to remain similar with the default lw value as well as to avoid “cluttering” with line widths thiner than the previous clipping boundary.